syntax = "proto3";

package example.complex;

message Dummy {
  int32 id = 1;
  string name = 2;
}

message Complex {
  Dummy one_dummy = 1;
  repeated Dummy multiple_dummies = 2;
}